ui/Screen/Org/Members.svelte

```diff
@@ -19,25 +17,55 @@
     display: flex;
     width: 100%;
     justify-content: space-between;
-    padding: 1.375rem 1.5rem;
+    padding: 1rem;
     align-items: center;
     min-width: 0;
   }
 
-  .member-name {
-    color: var(--color-foreground-level-6);
-  }
-
-  .member-address {
-    color: var(--color-foreground-level-5);
+  .member-details {
+    display: flex;
+    flex-direction: column;
+    align-self: center;
+    width: -webkit-fill-available;
+    min-width: 0;
   }
 </style>
 
 <div class="container">
   <List items={members} let:item={member} styleHoverState={false}>
     <div class="list-item">
-      <div class="member-name typo-text-bold">{member}</div>
-      <div class="member-address">{ellipsed(member)}</div>
+      {#if member.identity}
```

Comment:
We should keep the style like it's in the mockup -- we show the Ethereum address for unattested/unresolved members. And show the Avatar + name for attested ones:

<img width="646" alt="Screenshot 2021-06-16 at 11 47 53" src="https://user-images.githubusercontent.com/158411/122197649-b1e72b00-ce98-11eb-9aed-537215a24527.png">

Reply:
Fixed